images/aarch64/Ubuntu_16.04/shippable.yml

```diff
@@ -113,17 +113,15 @@ jobs:
       container: false
     steps:
       - IN: u16microbase_aarch64_dd_repo
-      - IN: u16_aarch64_dd_img
       - TASK:
           name: u16microbase_build
           runtime:
             options:
               env:
-                - IMG_BASE: "u16_aarch64_dd_img"
+                - REL_VER: "master"
```

Comment:
This is built on `drydock/aarch64_u16` still. See: https://github.com/dry-dock/aarch64_u16microbase/blob/master/Dockerfile#L1